ptbcontrib/ptb_sqlalchemy_jobstore/jobstore.py

```diff
@@ -101,15 +113,24 @@ def _reconstitute_job(self, job_state: bytes) -> APSJob:
         Args:
             job_state (:obj:`str`): String containing pickled job state.
         """
-        job = super()._reconstitute_job(job_state)  # pylint: disable=W0212
+        job: APSJob = super()._reconstitute_job(job_state)  # pylint: disable=W0212
 
         # Here we rebuild callback context for the job which
         # are going for execution.
-        tg_job = telegram.ext.Job(
-            callback=None,
-            name=job.args[0],
-            context=job.args[1],
+        name, data, chat_id, user_id, module = job.args
+        callback_module = import_module(module)
```

Comment:
I would suggest to evaluate if there are options that don't involve dynamic importing. AFAIK using `importlib` comes with several culprits and if it's not strictly necessary I like to avoid it. IIRC the `prepped_job` is just pickled and pickling the `callback` shouldn't be a problem, right?

Reply:
I had to use this approach since asynchronous callback can't be pickled. 

If there is any other option than dynamic importing I would prefer it as well.

Reply:
What you show here is a `Task` though.


```python
async def foo():
    pass

pickle.dumps(foo)
```

works perfectly fine and we only want to serialize the `ext.Job.callback`. BTW, if it helps to access the info of the job, you can is `ptb_job = telegram.ext.Job._from_aps_job(aps_job)` to reconstruct the PTB Job from an APS Job. We should probably make that method public in the main lib ... You can also directly use `ptb_job = aps_job.func.__self__`, which is what `_from_aps_job` does.

Reply:
So, I just tried to pickle `tg_job.callback` in L105, which is a coroutine function. After unpickling it, only the reference to function itself is what I get, e.g. `src.modules.do_something` at L120.

Reply:
Sorry, I don't understand what you're trying to say. Please double check that `tg_job.callback` is actually the jobs callback that you want to serialize. If it's not, the code needs to be reworked. I don't know what `src.module.do_something` is.

Reply:
Sorry, I wasn't clear. What I meant is, serializing `tg_job.callback` then deserializing it results in getting only that callback function `__method__` string, which points to the file with this function.

Reply:
please double check what kind of object `tg_job.callback` actually is.

Reply:
```
type(tg_job.callback)
<class 'function'>
tg_job.callback.__name__
'messages_scheduler'
tg_job.callback.__annotations__
{'context': telegram.ext._callbackcontext.CallbackContext[telegram.ext._extbot.ExtBot[None], typing.Dict[typing.Any, typing.Any], typing.Dict[typing.Any, typing.Any], typing.Dict[typing.Any, typing.Any]], 'return': None}
```

```
import inspect
print(inspect.getsource(tg_job.callback))
async def messages_scheduler(context: ContextTypes.DEFAULT_TYPE) -> None:
	...
```

Reply:
@Bibo-Joshi You were correct, there's no need to use dynamic imports indeed [`45aa87e` (#68)](https://github.com/python-telegram-bot/ptbcontrib/pull/68/commits/45aa87e95f0f0d127d087c249e461b1b5a46e24c)